packages/storybook/.storybook/main.ts

```diff
@@ -0,0 +1,96 @@
+import { StorybookConfig } from "@storybook/react-webpack5";
```

Comment:
I updated this file to `.ts` so that we could use this `StorybookConfig` type which has helped to ferret out a few incorrect webpack settings.